citus-ha.spec

```diff
@@ -33,8 +33,6 @@ make %{?_smp_mflags}
 %install
 PATH=%{pginstdir}/bin:$PATH
 %make_install
-%{__mkdir} -p %{buildroot}%{pginstdir}/bin
-%{__cp} /usr/pgsql-%{pgpackageversion}/bin/citusha %{buildroot}%{pginstdir}/bin/citusha
```

Comment:
`DESTDIR` got picked up here, therefore the binary is already in the correct place.